moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -222,15 +183,6 @@ const ProjectsListViewTable = () => {
     rowsPerPageOptions:
       PROJECT_LIST_VIEW_QUERY_CONFIG.pagination.rowsPerPageOptions,
   });
-  /*
-   * Store column configution before data change
```

Comment:
Not sure that this is needed without gqlabstract and GridTable. I don't see a regression, but I might not know the whole story here.

Reply:
I cannot remember exactly, but I think this had to do with requests in gqlabstract triggering multiple refreshes when it first loaded. so I agree that it probably isnt needed anymore